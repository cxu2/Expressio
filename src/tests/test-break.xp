~~ test semantic checking for `continue;`
~~ ./express.native -s ./tests/test-break.xp
~~ will print from 0-10, but break at 7, then prints(0,100,200) and -1 finally
main : -> int {
  int i;
  int j;
  for i = 0; i < 10; i = i + 1 {
    if(i==7){
      break;
    }
    print(i);
  }
  for j = 0;j<3;j=j+1{
    print(j*100);
  }
  print(-1);
  return 0;
}
