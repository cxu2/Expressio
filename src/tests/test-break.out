0
1
2
3
4
5
6
0
100
200
-1
